### DISCUSSION TITLE
What's the relationship between this project and workflow-core

### DISCUSSION THREAD

**Discussion by EvanCui** [elsa-workflows/elsa-core#3153] [Q&A] [answered]
It seems that the API spec is quite the same, so is there any relationship between these two projects?
https://github.com/danielgerlag/workflow-core

**Comment by EvanCui** [comment]
It will be great if you can reply.

**Comment by sfmskywalker** [comment, marked as answer]
Hi @EvanCui, 

Although the APIs look similar, there's no relationship between these two projects.
I took inspiration from various projects, including Workflow Core - which explains the similarity.

**Reply by EvanCui** [reply]
Thanks and well explained.
